lib/sinicum/imaging/imaging.rb

```diff
@@ -57,10 +57,14 @@ def file_converted
       end
 
       # The "final" file to be sent to the client
-      def file_rendered
-        @_file_rendered ||= File.join(config_data.file_dir, "/" + @renderer + "-" +
-          converter.config_hash + "-" + @image.fingerprint +
+      def file_rendered(srcset_affix = nil)
```

Comment:
This is still too complicated. The parameter should not be needed, since Imaging knows, which sourceset option you want.
And I don't think, that you need memoization with the param - the normal one should be fine.